src/main/utils/__tests__/metricsLogger.test.ts

```diff
@@ -1,24 +1,22 @@
 // @ts-nocheck
 
-import fetch from 'node-fetch'
+import { net } from 'electron'
 import metricsEvent from '../../../app/constants/metricsEvent'
 import metricsLogger from '../metricsLogger'
 import config from '../../config.json'
 
 jest.mock('electron', () => ({
   app: {
     getVersion: () => '1.0.0'
+  },
+  net: {
+    fetch: jest.fn()
   }
 }))
 
-jest.mock('node-fetch', () => ({
-  __esModule: true,
-  default: jest.fn()
-}))
-
 describe('metricsLogger', () => {
   const event = {
-    eventType: metricsEvent.downloadComplete,
+    eventType: metricsEvent.DownloadComplete,
```

Comment:
Isn't it 
```suggestion
    eventType: metricsEvent.DownloadComplete,
```
over in `src/app/constants/metricsEvent.js` so the casing should be the previous one?

Reply:
ugh, yes